lib/assets/chart_builder/main.js

```diff
@@ -1,358 +1,306 @@
+import * as Vue from "https://cdn.jsdelivr.net/npm/vue@3.2.26/dist/vue.esm-browser.prod.js";
+
 export function init(ctx, payload) {
   ctx.importCSS("main.css");
   ctx.importCSS("https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap");
 
-  ctx.root.innerHTML = `
-    <div class="app">
-      <div id="info-box" class="info-box"></div>
-      <div id="data-info-box" class="info-box"></div>
-      <div>
-        <div class="row">
-          <div class="box">
-            <div class="field">
-              <label class="input-label">Data</label>
-              <select class="input input--sm" name="data_variable">
-              </select>
-            </div>
-          </div>
-          <div class="box">
-            <div class="field">
-              <label class="input-label">Width</label>
-              <input class="input input--xs" name="width" type="number" />
-            </div>
-            <div class="field">
-              <label class="input-label">Height</label>
-              <input class="input input--xs" name="height" type="number" />
-            </div>
-          </div>
-          <div class="box">
-            <div class="field">
-              <label class="input-label">Title</label>
-              <input class="input input--md" name="chart_title" />
-            </div>
-          </div>
+  const app = Vue.createApp({
+    template: `
+      <div class="app">
+        <!-- Info Messages -->
+        <div id="info-box" class="info-box" v-if="missingDep">
```

Comment:
I don't remember much of Vue, but does the `id` stays global? Or does it only exist for the scope of this app? If is the former, then I would suggest to have a more "scoped" id, like `chart-builder-info-box` or something like that. WDYT?